src/style.css

```diff
@@ -0,0 +1,51 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+div {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  padding-top: 2px;
+  padding-left: 2px;
+}
+
+.stars:first-child {
+  padding-top: 2px;
+  padding-left: 2px;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  width: 15px;
+  height: 14px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 5px;
+}
+
+.stars--1 .stars__star:nth-child(-n+1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(-n+3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
Need to fix it
<img width="541" alt="image" src="https://github.com/mate-academy/layout_stars/assets/64750958/80636a88-5e2e-4129-a042-46a3fb0f73c3">